src/pd/timestamp.rs

```diff
@@ -132,10 +137,12 @@ impl<'a> Stream for TsoRequestStream<'a> {
         let pending_requests = self.pending_requests.clone();
         let mut pending_requests = pending_requests.borrow_mut();
         let mut count = 0;
```

Comment:
It looks that `count` is redundant as we can rely on `requests.len()`.

Reply:
> It looks that `count` is redundant as we can rely on `requests.len()`.

updated

Reply:
Please fix clippy